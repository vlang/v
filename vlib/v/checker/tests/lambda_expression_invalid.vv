fn main() {
	x := 5
	f := if x == 5 { |x| x - 5 } else { |x| x }
	println(f(x))
}
