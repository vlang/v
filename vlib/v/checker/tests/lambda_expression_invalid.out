vlib/v/checker/tests/lambda_expression_invalid.vv:3:19: error: lambda expressions are allowed only in places expecting function callbacks
    1 | fn main() {
    2 |     x := 5
    3 |     f := if x == 5 { |x| x - 5 } else { |x| x }
      |                      ~~~~~~~~~
    4 |     println(f(x))
    5 | }
